src/main/java/com/github/davidmoten/rtree/RTree.java

```diff
@@ -1,42 +1,29 @@
 package com.github.davidmoten.rtree;
 
-import static com.github.davidmoten.guavamini.Optional.absent;
-import static com.github.davidmoten.guavamini.Optional.of;
-import static com.github.davidmoten.rtree.geometry.Geometries.rectangle;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.List;
 
 import com.github.davidmoten.guavamini.Lists;
-import com.github.davidmoten.guavamini.Optional;
 import com.github.davidmoten.guavamini.annotations.VisibleForTesting;
-import com.github.davidmoten.rtree.geometry.Circle;
-import com.github.davidmoten.rtree.geometry.Geometry;
-import com.github.davidmoten.rtree.geometry.HasGeometry;
-import com.github.davidmoten.rtree.geometry.Intersects;
-import com.github.davidmoten.rtree.geometry.Line;
-import com.github.davidmoten.rtree.geometry.Point;
-import com.github.davidmoten.rtree.geometry.Rectangle;
+import com.github.davidmoten.rtree.geometry.*;
 import com.github.davidmoten.rtree.internal.Comparators;
 import com.github.davidmoten.rtree.internal.NodeAndEntries;
 import com.github.davidmoten.rtree.internal.operators.OperatorBoundedPriorityQueue;
-
 import rx.Observable;
 import rx.functions.Func1;
 import rx.functions.Func2;
 
+import java.util.*;
```

Comment:
no wildcard imports please

Reply:
Sorry, I used default setting of my ide. Do you have some style guide?
For a quick solution I increased the "class count to use import with '*'" to 10 in intelliJ

Reply:
no style guide, you can infer stuff like that from the code in front of you though.